src/_includes/macros/filters.html

```diff
@@ -0,0 +1,384 @@
+            <span class="expandable_cue-open">
+                Show
+              <span class="cf-icon cf-icon-plus-round"></span>
+            </span>
+            <span class="expandable_cue-close">
+                Hide
+                <span class="cf-icon cf-icon-minus-round"></span>
+            </span>
+        </span>
+    </button>
+    <div class="expandable_content">
+       {{ _filters_form(filters, query, doc_type, options) }}
+    </div>
+</div>
+
+{%- if options.show_current_filters -%}
+{{ active_filters_markup }}
+{%- endif -%}
+
+{% endmacro %}
+
+
+{# A helper macro for filters()
+   ========================================================================== #}
+{% macro _filters_form(filters, query, doc_type, options) %}
+
+<form class="{{ options.additional_classes }}"
+      method="get"
+{%- if options.action != '/' %}
+      action="{{ options.action }}">
+{%- else -%}
+      >
+{% endif %}
+    <div class="form-l form-l__float form-l__flush">
+{% for filter in filters %}
+
+    {# Category filters #}
+    {% if filter in ['blog_category', 'category'] %}
+        <div class="form-l_col form-l_col-1-3">
+            <div class="form-group">
+                <label class="form-label-header">
+                    Categories
+                </label>
+            {% if doc_type == 'posts' -%}
+                {% set categories = query.possible_values_for('blog_category')|sort(attribute='key') %}
+                {% set category_field = 'blog_category' %}
+            {%- else -%}
+                {% set categories = query.possible_values_for('category')|sort(attribute='key') %}
+                {% set category_field = 'category' %}
+            {% endif -%}
+            {% for category in categories -%}
+                <label class="form-group_item">
+                    {{ _filter_checkbox(category_field, category.key)|safe }}
+                </label>
+            {% endfor %}
+            </div>
+        </div>
+
+    {# Activity category filters #}
+    {% elif filter == 'activity_category' %}
+        <div class="form-l_col form-l_col-1-3">
+            <div class="form-group">
+                <label class="form-label-header">
+                    Category type
+                </label>
+            {% set categories =
+               query.possible_values_for('category')|sort(attribute='key') %}
+            {% for category in categories -%}
+                <label class="form-group_item">
+                    {{ _filter_checkbox('category', category.key)|safe }}
+                </label>
+            {% endfor %}
+            </div>
+        </div>
+
+    {# Calendar filters #}
+    {% elif filter == 'calendar' %}
+        <div class="form-l_col form-l_col-1-3">
+            <div class="form-group required-check-group">
+                <label class="form-label-header">
+                    Calendars
+                </label>
+                <label class="form-group_item">
```

Comment:
Should we move the directors into an array and loop through it rather than repeat code?